### PR TITLE
Update doco

### PR DESCRIPTION
**Pull request by mcsherrylabs**
Update README, Add LICENSE, add RELEASE so that it appears in the root folder on install.

### CODE REVIEW

File: src/universal/RELEASE [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+For a more detailed description of the functionality and configuration options see https://github.com/input-output-hk/etc-client/wiki
+
+Known Issues
+
+EC-269 JSON RPC network_version returns a value of 1 for morden network, correct version is 2
+
+EC-270 personal_importRawKey allows the same key to be imported several times
+
+EC-282 download gets stuck in a loop when downloading block headers from a single peer whichdoen't have those block headers
```
Comment (ntallar) [inline src/universal/RELEASE]: Minor typo comment, we should replace `whichdoen't` with `which doesn't`.

File: README.md [hunk 2]
```diff
@@ -1,46 +1,123 @@
-# etc-client
+# mantis - Scala client for Ethereum Classic
 
+*Mantis* is the name given to the new Ethereum Classic client produced by the Grothendieck Team.
```
Comment (AlanVerbner) [inline README.md]: Maybe a link to IOHK team page? https://iohk.io/projects/ethereum-classic/

File: README.md [hunk 3]
```diff
@@ -1,46 +1,123 @@
...
+
+This version of the code supports 
 
-[![CircleCI](https://circleci.com/gh/input-output-hk/etc-client/tree/master.svg?style=svg)](https://circleci.com/gh/input-output-hk/etc-client/tree/master)
+  - fast sync (download a recent state trie snapshot and all blocks, this is the default behaviour)
+  - regular sync (download and execute every transaction in every block in the chain, this can be very slow)
+  - bootstrap sync (download a database for *mantis* preloaded with a recent version of the block chain, recommended for testing) 
+  - JSON RPC API (useful for console and Mist integration)
+  - Morden testnet and private network
+  - `ethminer` miner integration (allows *mantis* to mine blocks with `ethminer`)
```
Comment (AlanVerbner) [inline README.md]: As there are several ethminer repos, maybe adding the link might be cool. I've used this one https://github.com/Genoil/cpp-ethereum

File: README.md [hunk 4]
```diff
@@ -1,46 +1,123 @@
...
+  
+This version has been tested on Windows 10, Linux and MacOS     
 
-[![Coverage Status](https://coveralls.io/repos/github/input-output-hk/etc-client/badge.svg?branch=master)](https://coveralls.io/github/input-output-hk/etc-client?branch=master)
+For more details on configuration and functionality check out our [wiki](https://github.com/input-output-hk/etc-client/wiki)
 
+### Download the client and bootstrap files
```
Comment (AlanVerbner) [inline README.md]: Maybe a link to bootstrap files page?

File: README.md [hunk 5]
```diff
@@ -1,46 +1,125 @@
...
-## A Scala based client for Ethereum Classic 
+The bootstrap database files can be downloaded from [here](https://github.com/input-output-hk/etc-client/wiki/Bootstrap-Database-Download-Links)
+    
+### Building the client 
 
-### Milestone 3 & 4 - JSON RPC and Miner Integration
+#### Prerequisites to build
 
-**This version of the code supports almost all the functionality for a first release.**
+- JDK 1.8 (download from [java.com](http://www.java.com))
```
Comment (AlanVerbner) [inline README.md]: Maybe link should be pointing to https://www.java.com/en/download/

**Pull request merged**